File: src/main/java/by/devpav/serfor/resources/VirtualDirectoryResource.java
package by.devpav.serfor.resources;

import by.devpav.serfor.domain.dtos.VirtualDirectoryDTO;
import org.springframework.http.ResponseEntity;

import java.util.List;

public interface VirtualDirectoryResource {

    public ResponseEntity<List<VirtualDirectoryDTO>> getByRealmName(String realmName);

}

File: src/main/java/by/devpav/serfor/resources/VirtualDirectoryResource.kt
package by.devpav.serfor.resources

import by.devpav.serfor.domain.dtos.VirtualDirectoryDTO
import org.springframework.http.ResponseEntity

interface VirtualDirectoryResource {

    fun getByRealmName(realmName: String): ResponseEntity<List<VirtualDirectoryDTO>>

}
